improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -445,6 +445,8 @@ def set_up_percentile_cube(
     cube = set_up_variable_cube(data, realizations=percentiles, **kwargs,)
     cube.coord("realization").rename("percentile")
     cube.coord("percentile").units = Unit("%")
+    if len(percentiles) == 1:
+        cube = next(cube.slices_over("percentile"))
```

Comment:
```suggestion
        cube = iris.util.squeeze(cube)
```
Although I would have expected this statement to be in the `set_up_variable_cube` function. Why is it done here?

Reply:
Two things going on here:

1.  Why am I doing the squeezing here?  This is because realization (which is what's set up in `set_up_variable_cube`) always wants to be a dimension, even if single-valued.  This is consistent with the enforcement in `improver.standardise`.
1.  Why am I slicing not squeezing?  Because of the wind downscaling unit tests, many of which use a single grid point in the x- and y- dimensions.  We want to keep these as scalar dimensions.  So I make the setup functions here slice only over the coordinates they've created, which keeps them from having unintended side effects.

Reply:
Ok